python/mxnet/ndarray/ndarray.py

```diff
@@ -2518,8 +2518,8 @@ def arange(start, stop=None, step=1.0, repeat=1, ctx=None, dtype=mx_real_t):
     """
     if ctx is None:
         ctx = current_context()
-    return _internal._arange(start=start, stop=stop, step=step, repeat=repeat,
-                             dtype=dtype, ctx=str(ctx))
+    return _internal._arange(start=start, stop=stop, step=step, infer_range=infer_range,
+                             repeat=repeat, dtype=dtype, ctx=str(ctx))
```

Comment:
nit: infer_range and repeat keyword arguments swapped place (not that it matters)